src/DynamoCoreWpf/Views/Core/DynamoView.xaml

```diff
@@ -1478,21 +1490,21 @@
 
         </Border>
 
-        <!--Collapsed Sidebar-->
+        <!--Collapsed Left Sidebar-->
         <StackPanel Grid.Row="2"
                     Grid.Column="2"
                     Orientation="Vertical"
-                    Name="collapsedSidebar"
+                    Name="collapsedLibrarySidebar"
```

Comment:
I wonder if we should consider these names public APIs?

Reply:
You mean so people could access these without looking at our code?